src/consensus/consensus.h

```diff
@@ -23,7 +23,8 @@ static const unsigned int MAX_BLOCK_SIZE = 2000000;
 /** The maximum allowed number of signature check operations in a block (network rule) */
 static const unsigned int MAX_BLOCK_SIGOPS = 20000;
 /** The maximum size of a transaction (network rule) */
-static const unsigned int MAX_TX_SIZE = 100000;
+static const unsigned int MAX_TX_SIZE_BEFORE_SAPLING = 100000;
+static const unsigned int MAX_TX_SIZE = MAX_BLOCK_SIZE;
```

Comment:
`MAX_TX_SIZE` should be `MAX_TX_SIZE_AFTER_SAPLING` for explicitness and so that we have to touch every bit of code using the constant.

Reply:
Maybe, given that there will be more upgrades,  `MAX_TX_SIZE_SAPLING` is better without the AFTER ?